github/pull_request.rb

```diff
@@ -55,14 +55,22 @@ def check_commits_style
       if (commit.commit.message.lines.first =~ /\A(fixes|refs) #\d+(, ?#\d+)*(:| -) .*\Z/i) != 0
         warnings += "  * #{commit.sha} must be in the format ```fixes #redmine_number - brief description```\n"
       end
+      if commit.commit.message.lines.first.size > 65
+        warnings += "  * length of the first commit message line for #{commit.sha} exceeds 65 characters\n"
+      end
+      commit.commit.message.lines.each do |line|
+        if line.size > 72 && line =~ /^\s{4,}/
```

Comment:
Does this only add a warning if the line is starting four or more spaces?  Shouldn't it be the reverse?